pkgs/development/compilers/minimacy/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib, stdenv, fetchFromGitHub, makeBinaryWrapper
+, alsaLib, libX11, libXext, libGL, libGLU
+}:
+
+stdenv.mkDerivation rec {
+  pname = "minimacy";
+  version = "0.6.2";
+
+  src = fetchFromGitHub {
+    owner = "ambermind";
+    repo = pname;
+    rev =  version;
+    sha256 = "i0Z1UKApX+elHmFgujwjiF7k6OmtFF37HJim464OMfU=";
+  };
+
+  nativeBuildInputs = [ makeBinaryWrapper ];
+
+  buildInputs = [ libGL libGLU ] ++ lib.optionals stdenv.isLinux [ alsaLib libX11 libXext ];
+
+  enableParallelBuilding = true;
+
+  NIX_CFLAGS_COMPILE = "-Wno-unused-result";
+
+  patchPhase = lib.optional stdenv.isDarwin
+    ''
+      substituteInPlace ./macos/cmdline/makefile \
+        --replace gcc cc
```

Comment:
`$(CC)` is more proper, as for cross compilation scenarios, the compiler could be prefixed with the target name resulting in something like `riscv64-unknown-gnu-gcc`

Reply:
Thanks. I pushed a new version that runs `make nox CC=cc` on darwin instead of running a patchPhase.

Reply:
Thanks. I pushed a new version that runs `make nox CC=cc` on darwin instead of running a patchPhase. I am just trying to get around the [hardcoded](https://github.com/ambermind/minimacy/blob/master/macos/cmdline/makefile#L2) default so I don't have to pull in an alternate stdenv for darwin.